module/VuFind/src/VuFind/Reserves/CsvReader.php

```diff
@@ -162,7 +162,7 @@ protected function loadFile($fn)
             throw new \Exception("Could not open $fn!");
         }
         $lineNo = $goodLines = 0;
-        while ($line = fgetcsv($fh, 0, $this->delimiter)) {
+        while ($line = fgetcsv($fh, 0, $this->delimiter, escape: '\\')) {
```

Comment:
This appears to be an accidental change unrelated to the rest of this PR.

```suggestion
        while ($line = fgetcsv($fh, 0, $this->delimiter)) {
```